src/modules/mail/mail.controller.ts

```diff
@@ -0,0 +1,12 @@
+import { Controller, Param, Post } from '@nestjs/common';
+import { MailerService } from './mail.service';
+
+@Controller('mailer')
+export class MailerController {
+  constructor(private readonly mailerService: MailerService) {}
+
+  @Post('/:id')
+  async sendMail(@Param('id') id: string) {
+    return await this.mailerService.sendReviewMail(id);
+  }
```

Comment:
Does this endpoint necessary? Shouldn't it be sent automatically after checkout?

Reply:
it is not needed, i used it to check services